modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -0,0 +1,26 @@
+= Mule App Deployment Model
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, architecture
+
+A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+
+* Running multiple applications side-by-side
```

Comment:
What does "Running multiple applications side-by-side mean and why is this an advantage?

Reply:
In this case, I would change it to _Running independently multiple applications side-by-side within the same Mule container. @aiannucci @marianogonzalez Could you reconfirm the benefit/advantage of this? e.g: Running 2 apps within same Mule container, vs running each app in different Mule container. Is it regarding performance?

Reply:
Updated.
Added this extra line _If you have to build an application with some complexity, you can divide it by creating several apps with their specific logic and then deploy all the apps within the same Mule container._